_docs/policies/README.md

```diff
@@ -0,0 +1,38 @@
+# Foundational Principles for Policies
+
+**Purpose**: Policies establishes the fundamental rules that govern the creation, amendment, and enforcement of all actions within the status-go project. These policies reflect our core 
+values of inclusivity, transparency, and consensus-driven decision-making while defining enforceable rules that guide status-go contributions. Policies are not merely guidelines but are to be 
+respected and adhered to, ensuring alignment across the community.
+
+## 1. Inclusivity
+
+Every member of the status-go community must have an equal opportunity to participate in the creation and review of policies. Contributions from all backgrounds and perspectives are encouraged and 
+respected. The process must ensure that:
+
+- **Open participation**: Any individual or group who interacts with or contributes to the project has the right to voice their opinions and suggestions.
+- **Accessibility**: Policies must be written in clear, accessible language, and the decision-making process should be easy to understand for everyone involved.
+
+## 2. Transparency
+
+All discussions, decisions, and processes related to policy creation must be fully transparent. This includes:
+
+- **Open documentation**: Discussions leading to policy decisions must be documented and made publicly available. This ensures that anyone can review the rationale behind a decision.
+- **Clear communication**: Changes to policies, including their rationale, must be clearly communicated to the community through appropriate channels.
+
+## 3. Consensus
+
+Policies must be created and modified through a consensus-based approach, ensuring that the community agrees on how to move forward. The following principles guide consensus building:
+
+- **Collaborative discussion**: Proposals must be discussed openly, with all points of view considered.
+- **Dispute resolution**: If disagreements arise, efforts must be made to find a compromise that balances the needs of all stakeholders.
+- **Final agreement**: Policies should be approved by a clear consensus, meaning that while not everyone may agree 100%, all should feel their voices were heard and respected, and the final decision 
+reflects the community’s general will.
+
+## 4. Enforceability and Respect for Policies
+
+- **Key stakeholder approval**: The guild does not have unilateral enforcement power. Instead, policies require explicit recorded approval from all key stakeholders before becoming enforceable. This 
+includes team leads, the status-go Guild, and other relevant parties.
+- **Respect and adherence**: Policies are not optional guidelines. Once approved, they are enforceable rules that all contributors to the status-go project are expected to respect. This ensures 
```

Comment:
It's a bit unclear to me how the enforceability may work on a daily basis and how we can be sure that the policies are fully respected. Are reviewers the ones who are responsible for verification of given part of code regarding policies? Different people have different levels of understanding and can enforce them to different degrees. In practice, old habits also do well because they allow to move forward easily, even if it is a short-sighted action. Adjusting to new arrangements often means spending extra time to understand them and leaving the comfort zone, both for pr author and reviewers. In practice, this results in incomplete implementation of the arrangements. The question is whether we have any specific arrangements on how to counteract this

Reply:
@micieslak 

TLDR; Policies should not define HOW they're enforced, but define WHAT we agree to enforce.

-----

> It's a bit unclear to me how the enforceability may work on a daily basis

It depends on the certain policy. But in most cases it should be automated.

For example, "pull requests must have > 50% diff code coverage". 
Obviously, it can't be checked by reviewers and should be automated (which we did).

Another example, "breaking changes policy", is not easy to automate, so indeed it would be mostly one developers and reviewers shoulders.

Reply:
Thank you @micieslak, I have personally struggled with the concept of enforceability and where we draw the line between guidelines and rules. I don't like rules, and I don't like arbitrary rules.

The points you make are all reasonable and thoughtful, and I agree with everything you write, these are legitimate everyday scenarios. I believe that some cultural changes may be necessary from all `status-go` contributors and this will in some cases cause some slowing down of development speed. This is not a bad thing, in many cases it should reduce the overall development and QA time spent on features.

My opinion is that policies should be a social contract that `status-go` contributors agree to, this is why I have included a lot of detail about the importance of consensus and inclusivity. Policies should be introduced for things that can not be automated but are still essential for the long term health of the repo. The examples Igor make are good ones to highlight, as they show a distinction between what is automatable and what is not, yet both are needed for the long term health of the repo.

> old habits also do well because they allow to move forward easily, even if it is a short-sighted action.

This particularly is an incredibly important point Michał, and it is in fact a major motivator for introducing a policy process that requires a high level of consensus. By engaging all contributors and leads my personal hope is that short-sighted behaviour will be seen as somewhat selfish and generally detrimental to the work of our fellow contributors, even across teams and specialities.

By requiring a very high level of consensus for any policy we can build mutually approved guidelines that will ensure that contributors from many teams and backgrounds have clear expectations and an even playing field.